mne/gui/_coreg.py

```diff
@@ -527,10 +527,29 @@ def _subject_changed(self, changed=None):
 
     @observe("_lock_fids")
     def _lock_fids_changed(self, change=None):
-        locked_widgets = ["sX", "sY", "sZ", "tX", "tY", "tZ",
-                          "rX", "rY", "rZ", "project_eeg",
-                          "fit_fiducials", "fit_icp",
-                          "save_mri_fids"]
+        locked_widgets = [
+            # MRI fiducials
+            "save_mri_fids",
+            # View options
+            "project_eeg", "helmet", "head_opacity", "high_res_head",
+            # Digitization source
+            "info_file", "grow_hair", "omit_distance", "omit", "reset_omit",
+            # Scaling
+            "scaling_mode", "sX", "sY", "sZ",
+            # Transformation
+            "tX", "tY", "tZ",
+            "rX", "rY", "rZ",
+            # Fitting buttons
+            "fit_fiducials", "fit_icp",
+            # Transformation I/O
+            "save_trans", "load_trans",
+            "reset_trans",
+            # ICP
+            "icp_n_iterations", "icp_fid_match", "reset_fitting_options",
+            # Weights
+            "hsp_weight", "eeg_weight", "hpi_weight",
+            "lpa_weight", "nasion_weight", "rpa_weight",
+        ]
```

Comment:
More so if the spin boxes' labels and layout can be disabled too as you suggested

Reply:
It's really just an intermediate step because I looked at how to make an entire layout / groupbox inactive and it's not trivial due to amount of nesting we use. We can discuss this later. But I think this PR is already an improvement, albeit not the optimal solution